.goreleaser.snapshot.yml

```diff
@@ -92,3 +92,35 @@ signs:
       - '{{ .Env.SIGN_KEY_NAME }}'
       - '${artifact}'
     stdin_file: /tmp/gpg_passphrase
+
+snapcrafts:
+  - id: stack
+    builds:
+      - cli
+      - stack
+    name: ttn-lw-stack
+    summary: The Things Stack for LoRaWAN
+    description: The Things Stack for LoRaWAN
+    grade: stable
+    confinement: strict
+    publish: false
```

Comment:
Since we don't publish the snap nor save the artifacts I would say it's better to not build the snaps in snapshots.

We generate snapshots for every merge commit, but we wouldn't save these snaps, which is wasteful.

Reply:
I didn't know that it would be generated after every merge commit, then it is indeed inappropriate to have the `snapcraft` options in here.